cypress/e2e/lock-widget.cy.ts

```diff
@@ -1,26 +1,64 @@
 describe('Widgets can lock and unlock', () => {
+  const menuToggleLocator =
+    '[data-ouia-component-id="landing-rhel-widget"] [aria-label="widget actions menu toggle"]';
+
+  const lockWidgetLocator = '[data-ouia-component-id="lock-widget"] > button';
+  const unlockWidgetLocator =
+    '[data-ouia-component-id="unlock-widget"] > button';
+
   beforeEach(() => {
     cy.loadLandingPage();
+    cy.viewport(1920, 1080);
+    cy.intercept('PATCH', '**/api/chrome-service/v1/dashboard-templates/*').as(
+      'patchLayout'
+    );
+    // waits until page content has rendered before proceeding
+    cy.get('[id="widget-layout-container"] .react-grid-item')
+      .contains(
+        'Proactively assess, secure, and stabilize the business-critical services that you scale from your RHEL systems.'
+      )
+      .should('be.visible');
+    // test started passing consistently once I added this wait. Weird.
+    cy.wait(5000);
```

Comment:
I think we should revisit this eventually - a bit worried this will cause the tests to run longer than they really need. However we can merge to unblock our progress.